src/types/condition_opcodes.py

```diff
@@ -3,18 +3,33 @@
 
 
 class ConditionOpcode(bytes, enum.Enum):
-    UNKNOWN = bytes([49])
-    AGG_SIG = bytes([50])
+    # UNKNOWN is ascii "0"
+    UNKNOWN = bytes([48])
+
+    # AGG_SIG is ascii "1"
+
+    # signature opcodes
+    AGG_SIG = bytes([49])
+    AGG_SIG_ME = bytes([50])
+
+    # creation opcodes
     CREATE_COIN = bytes([51])
-    ASSERT_ANNOUNCEMENT = bytes([52])
-    ASSERT_MY_COIN_ID = bytes([53])
-    ASSERT_RELATIVE_TIME_EXCEEDS = bytes([54])
-    ASSERT_BLOCK_INDEX_EXCEEDS = bytes([55])
-    ASSERT_BLOCK_AGE_EXCEEDS = bytes([56])
-    AGG_SIG_ME = bytes([57])
-    ASSERT_FEE = bytes([58])
-    ASSERT_TIME_EXCEEDS = bytes([59])
-    CREATE_ANNOUNCEMENT = bytes([60])
+    CREATE_ANNOUNCEMENT = bytes([52])
+
+    # assertions: coins & announcements
+    ASSERT_ANNOUNCEMENT = bytes([53])
+    ASSERT_MY_COIN_ID = bytes([54])
+
+    # wall-clock time
+    ASSERT_ABSOLUTE_TIME_EXCEEDS = bytes([55])
+    ASSERT_RELATIVE_TIME_EXCEEDS = bytes([56])
+
+    # block index
+    ASSERT_BLOCK_INDEX_EXCEEDS = bytes([57])
+    ASSERT_BLOCK_AGE_EXCEEDS = bytes([58])
```

Comment:
block age is confusing IMO

Reply:
How about unifying, so `ASSERT_ABSOLUTE_BLOCK_EXCEEDS` and `ASSERT_RELATIVE_BLOCK_EXCEEDS` (or `BLOCK_INDEX` but that's a bit long). @wbendick comments?

Reply:
Possible 'logical' names are ASSERT_BLOCK_HEIGHT_EXCEEDS, ASSERT_COIN_HEIGHT_EXCEEDS, ASSERT_TIME_EXCEEDS and ASSERT_COIN_AGE_EXCEEDS

Reply:
It's a 2x2 grid of "wall clock/block index" by "absolute/relative". It'd be nice if we could come up with some naming for that. `AGE` suggests relative, but it doesn't disambiguate between wall/block. Maybe `SECONDS` for wall clock and `BLOCK` for block. And `AGE` for relative vs `NOW` for absolute. So `ASSERT_SECONDS_AGE`, `ASSERT_SECONDS_NOW`, `ASSERT_BLOCK_AGE`, `ASSERT_BLOCK_NOW`. I dunno.

Reply:
Or we could be extra cute and go with `CLOCK` and `BLOCK`.

Reply:
Maybe we could go with AGE/NOW and SECONDS/HEIGHT. So that would be ASSERT_SECONDS_AGE, ASSERT_SECONDS_NOW, ASSERT_HEIGHT_AGE, and ASSERT_HEIGHT_NOW. These seem to imply >= instead of >, not sure what the underlying conditions currently do. Allowing zero with >= seems a bit more natural.